transform/models/marts/geo_reference/_geo_reference__models.yml

```diff
@@ -0,0 +1,88 @@
+version: 2
+
+models:
+  - name: dim_geo_reference_footprints_blocks__joined
+    description: |
+      This data table is a join of the TIGER shapefile, Blocks,
+      with the Microsoft Building Footprints data for the state of CA.
+    columns:
+      - name: release
+        description: The version of the data
+      - name: capture_dates_range
+        description: Each building footprint has a capture date tag from 2019-2020
+      - name: county_fp
```

Comment:
I don't love `fp` and `ce` as suffixes: I think `fp` stands for FIPS and `CE` stands for Census? Neither seem like particularly useful abbreviations, and are instead probably legacy nonsense from dealing with shapefiles. I think I'd prefer `county_fips`, `tract`, and `block`

Reply:
Makes sense!